src/clarity-angular/emphasis/alert/_alert.clarity.scss

```diff
@@ -81,30 +77,70 @@ $clr-alert-colors: (
     @else {
         border: none;
     }
+
+    .alert-icon {
+        color: map-get($propMap, icon-color)
+    }
 }
 
 @include exports('alert.clarity') {
-    %icon-styles {
+    $alertIconWidth: $clr_baselineRem_1 + 1;
```

Comment:
Because the icons are a very explicit size in the specs and due to the nature of needing to have them centered in the min-height of the alert, I had to do some pixel pushing. You'll notice that the icon itself is larger than its wrapper by a decent amount. The sizing, padding, and negative margins help keep everything lined up.